src/services/auth/auth.js

```diff
@@ -0,0 +1,66 @@
+import axios from 'axios';
+
+import { clearAuthTokens } from '../token';
+
+import config from '../../config';
+
+/**
+ * Maps to token object.
+ *
+ * @param {Object} response
+ *
```

Comment:
I don't think an empty line is necessary before a return tag in JSDoc.
Let's remove this empty line, and make the JSDoc consent within the repo.